packages/migrate/migrations/svelte-5/index.js

```diff
@@ -56,6 +58,42 @@ export async function migrate() {
 		}
 	}
 
+	const kit_dep = pkg.devDependencies?.['@sveltejs/kit'] ?? pkg.dependencies?.['@sveltejs/kit'];
+	if (kit_dep && semver.validRange(kit_dep) && semver.gtr('2.0.0', kit_dep)) {
```

Comment:
it's a little surprising to me to see `2 > kit_dep` vs `kit_dep < 2`

Reply:
yoda-speak - it's done like that in all other migration, I don't care honestly.